catalog/resourcemanagement/resourcemanagement-app/pom.xml

```diff
@@ -80,7 +80,6 @@
             <plugin>
                 <groupId>org.apache.karaf.tooling</groupId>
                 <artifactId>karaf-maven-plugin</artifactId>
-                <version>${karaf.version}</version>
```

Comment:
The version for the `karaf-mavin-plugin` is defined in the`pluginManagement` section of the root pom: https://github.com/codice/ddf/blob/master/pom.xml#L396. I wasn't sure what the beast practices are when it comes to `pluginManagement` (as opposed to `dependencyManagement`), so I just made these `karaf-mavin-plugin` lines consistent with the rest of the poms in DDF.